torch/csrc/jit/mobile/function.cpp

```diff
@@ -214,6 +218,26 @@ c10::optional<std::function<void(Stack&)>> makeOperatorFunction(
   return fn;
 }
 
+Function& Function::registerFunc(
+    const std::string qualified_name,
+    const std::vector<Instruction>& instructions,
+    const std::vector<c10::IValue> constants,
+    const std::vector<c10::TypePtr> types,
+    const google::int64 register_size) {
+  static Function func = Function(c10::QualifiedName(qualified_name));
```

Comment:
`static` here may still cause potential issue of adding instructions of different upgraders to the same function.

Reply:
My changes are not updated...sorry about that and I push a new commit.